File: Daily Challenge/Java/Binary Growing Sequence Or Not 7.7.2025.java
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt(), a[] = new int[n];
        for(int i = 0; i < n; i++) a[i] = sc.nextInt();
        for(int i = 0; i < n - 1; i++) {
            if((a[i] & a[i + 1]) != a[i]) {
                System.out.print("NO");
                return;
            }
        }
        System.out.print("YES");
    }
}

File: Daily Challenge/C/Binary Growing Sequence Or Not 7.7.2025.c
#include <stdio.h>
int main() {
    int n, a[1000], i;
    scanf("%d", &n);
    for(i = 0; i < n; i++) scanf("%d", &a[i]);
    for(i = 0; i < n - 1; i++)
        if((a[i] & a[i + 1]) != a[i]) {
            printf("NO");
            return 0;
        }
    printf("YES");
    return 0;
}